Added Spanish Language (es_es)

FrannDzs opened the pull request:
Hi, after some time away from Github I corrected the branch and re-did everything from scratch. Cheers!

FrannDzs committed:
Added Spanish (es_es) language

FrannDzs committed:
Create es_es

FrannDzs committed:
Delete es_es

FrannDzs committed:
Create es_es.json

FrannDzs committed:
Update and rename es_es.json to asphalt.txt

FrannDzs committed:
Update asphalt.txt

FrannDzs committed:
Create automaticlubricator.txt

FrannDzs committed:
Create coker.txt

FrannDzs committed:
Rename coker.txt to cokerunit.txt

FrannDzs committed:
Create distillationtower.txt

FrannDzs committed:
Create flarestack.txt

FrannDzs committed:
Update flarestack.txt

FrannDzs committed:
Create UNITI

FrannDzs committed:
Rename UNITI to hydrotreater.txt

FrannDzs committed:
Create lubricant.txt

FrannDzs committed:
Create napalm.txt

FrannDzs committed:
Create portablegenerator.txt

FrannDzs committed:
Create projector.txt

FrannDzs committed:
Create pumpjack.txt

FrannDzs committed:
Create speedboat.txt

FrannDzs committed:
Update credits.md

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Merge branch 'TwistedGate:1.16.5' into 1.16.5

paladini-maker committed:
Update es_es.json

Some improvements and fixes

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Update asphalt.txt

TwistedGate commented on src/main/resources/assets/immersivepetroleum/lang/es_es.json:
```diff
@@ -118,12 +118,12 @@
 	"_comment": "################## IE-Manual ##################",
 	
-	"manual.immersivepetroleum.main": "Procesado de petróleo",
+	"manual.immersivepetroleum.main": "§lProcesado de petróleo§r",
```
Not a big fan of the category text being **fat**

Any specific reason for that?

FrannDzs replied:
Hello @TwistedGate , yes, I am in responsible for the Spanish translation of immersive engineering, and the entries in the manual have been redone in bold, and to match the rest is why I put it here too, although it is not finished yet...

FrannDzs replied:
@TwistedGate Should I fix it or can I keep it that way?

TwistedGate replied:
> 
> 
> Hello @TwistedGate , yes, I am in responsible for the Spanish translation of immersive engineering, and the entries in the manual have been redone in bold, and to match the rest is why I put it here too, although it is not finished yet...

Alrighty

> 
> 
> @TwistedGate Should I fix it or can I keep it that way?

Keep it

TwistedGate commented on src/main/resources/assets/immersivepetroleum/lang/es_es.json:
```diff
@@ -118,12 +118,12 @@
 	
-	"ie.manual.entry.reservoirs.title": "Depósito de fluídos",
-	"ie.manual.entry.reservoirs.subtitle": "Oro negro",
+	"ie.manual.entry.reservoirs.title": "§lDepósito de fluídos§r",
+	"ie.manual.entry.reservoirs.subtitle": "§oOro negro§r",
```
Same with these two

TwistedGate commented on src/main/resources/assets/immersivepetroleum/manual/es_es/asphalt.txt:
```diff
@@ -1,3 +1,3 @@
-Hormigón de asfalto
-Material para carreteras
-<&asphalt0>Hormigón de asfalto, unidos por el pegajoso betún, es una alternativa de color más oscuro al Hormigón normal. Puede producirse utilizando los subproductos del procesamiento del petróleo crudo en la <link;immersivepetroleum:distillationtower;§n§oTorre de §n§oDestilación§r>.
+§lHormigón de asfalto§r
+§oMaterial para carreteras§r
```
And these..

paladini-maker committed:
Updated spanish language

- Some capitalization fixes
- Now the immersive petroleum manual entries are in bold, like the immersive engineering content.
- Searched the internet for information about multiblocks to call them as they are in real life.

paladini-maker committed:
Update distillationtower.txt

paladini-maker committed:
Updated spanish language

- Added spanish language variations
